src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1354,13 +1354,17 @@ private void startContainer() {
     private Process getRunProcess(String command) throws IOException {
         ProcessBuilder processBuilder = new ProcessBuilder();
         processBuilder.command(getCommandTokens(command));
+        Map<String, String> env = processBuilder.environment();
         if (!OSUtil.isLinux()){
-            Map<String, String> env = processBuilder.environment();
             if (!env.keySet().contains("DOCKER_BUILDKIT")) { // don't touch if already set
                 env.put("DOCKER_BUILDKIT", "0"); // must set 0 on Windows VMs
                 debug("Generating environment for docker build & run: DOCKER_BUILDKIT=0");
             }
         }
+        if (!env.keySet().contains("DOCKER_SCAN_SUGGEST")) { // don't touch if already set
+            env.put("DOCKER_SCAN_SUGGEST", "false"); // Disable stderr contents from showing as error message
```

Comment:
Just to be clear, the intention is not to disable all stderr contents from being displayed as an error message, right?
Just the "Synk tests" message?

Reply:
Right, the general idea of propagating stderr to error logging msgs remains intact...we just stopped this particular msg getting written to stderr by `docker build`.    Yes, it seems to be also doing the same for `docker run` but the DOCKER_BUILDKIT one did too so I guess that's OK.